### DISCUSSION TITLE
Find hidden animals

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#868] [Galaxy Trivia] [answered]
```
ttr
pnd
frrt
s ln
hdghg
chth
tdpl
plr br
lprd
grff
```

*Answer with emoji (unleash your creativity)*

**Comment by PatilHarshh** [comment]
ttr:  🐯Tiger 
pnd:  🐼 Panda
frrt:  🐹Ferret
s ln:🦭 Sealion
hdghg: 
chth: 🐆Cheetah
tdpl: 🐸  Toadstool or Tadpole 
plr br:  🐻 Polar Bear
lprd:  🐆 Leopard
grff: 🦒Giraffe

**Comment by peterbaccaro** [comment, marked as answer]
ttr: 🐟 **T**e**tr**a
pnd: 🐼 **P**a**nd**a
frrt: 🐹 **F**e**rr**e**t**
s ln: 🦭 **S**ea **L**io**n**
hdghg: 🦔 **H**e**dg**e**h**og
chth: 🐆 **Ch**ee**t**a**h**
tdpl: 🐸 **T**a**dp**o**l**e
plr br: 🐻‍❄️ **P**o**l**a**r** **B**ea**r**
lprd: 🐆 **L**eo**p**a**rd**
grff: 🦒 **G**i**r**a**ff**e

**Comment by ParanoidUser** [comment]
**Hint:** *ttr* eat mostly 🐟, 🐸, 🐢, 🦀

**Reply by peterbaccaro** [reply]
ttr: 🦦 O**tt**e**r**

**Reply by PatilHarshh** [reply]
otter: 🦦
